model_building/wrapper_experiment_configuration.py

```diff
@@ -262,7 +297,8 @@ def print_model(self):
         """
         Print the representation of the generated model
         """
-        return "".join(("Features selected with SFS: ", str(self.get_regressor().aml_features), "\n",
+        #return "".join(("Features selected with SFS: ", str(self.get_regressor().aml_features), "\n",
```

Comment:
!

Reply:
Removed as the other comments, in my defense I can assure you there were many other